torch/linalg/__init__.py

```diff
@@ -953,6 +1076,84 @@
     tensor(5.4345)
 """)
 
+matrix_norm = _add_docstr(_linalg.linalg_matrix_norm, r"""
+linalg.matrix_norm(A, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
```

Comment:
Similarly to the `ord=None` vs. `ord='fro'` discussion below, should `dim` just have a default value of `(-1, -2)` instead of being `None`?

Reply:
aye!

Reply:
FYI: The default is (-2, -1) which is a different matrix then (-1, -2)

Reply:
yeah, sorry, it should be `(-2, -1)`

Reply:
default changed.

Reply:
Excuse me, you're correct @heitorschueroff, my mistake